LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Mattias Erming
```

Comment:
Oh boy, that moment at the end of a talk when you hope no one would raise their hands when you ask "Any questions?"

I don't know. This is a tricky question. At first I was like, sure why not, then after reading [this](http://crypti.cc/blog/2015/02/17/contributor-license-agreement/), I'm like, heck, that's why not. Honestly, we **do not** want a CLA, that would just get in the way of enhancing that project (I know of one project I contribute to that requires to sign a CLA, and boy it's tedious for everyone).

This seems like we would need a keen eye on this. I'm sure she is very busy, but if @wseltzer happens to receive notifications, and would be okay to help us a bit, that would be great :-)
@wseltzer, essentially, this is an Free and Open Source Software project under the MIT license and we get *a lot* of contributions. What would be the best thing to type here? Of course it's always nice when we give the right credit to the right people, but on the other hand if that forces us to get agreement from all contributors if we decide to change license), that would be hard to organize.

@JocelynDelalande, what I am sure about though, is that I want this PR to make it into master quickly :-) Can I get your +1 *and* continue the discussion to reach to a solution even if it has to be done post-merged, in another PR?